packages/web-react/src/components/TextField/demo/index.tsx

```diff
@@ -46,15 +53,12 @@ ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
       <DocsSection title="Fluid">
         <TextFieldFluid />
       </DocsSection>
-      <DocsSection title="Size">
+      <DocsSection title="Input Sizes">
```

Comment:
```suggestion
      <DocsSection title="Input Width">
```

Reply:
Part of another old PR: https://github.com/lmc-eu/spirit-design-system/pull/1269/files 🙈